workflow/rules/common.smk

```diff
@@ -163,24 +163,39 @@ def get_cnv_segments(wildcards):
     raise NotImplementedError(f"not implemented for caller {wildcards.caller}")
 
 
-def get_germline_vcf(wildcards: Wildcards) -> List[Union[str, Path]]:
+def get_germline_vcf(wildcards: Wildcards) -> Union[List, Path]:
```

Comment:
A better type hint would be to define a type that is an empty list of string, since this is the only type of list that should be returned from this function.

Reply:
Addressed this in a68ed7d.